server/controllers/request.js

```diff
@@ -0,0 +1,35 @@
+const Request = require("../models/Request");
+const express = require("express");
+const router = express.Router();
+
+// @desc  GET : list of requests for logged in user
+// @route GET
+
+const getRequests = async (req, res) => {
+  const requests = await Request.find({ userId: req.user.id });
+  res.send(requests);
+};
+
+// @desc  POST : Create a new request
+// @route POST
+
+const createRequest = async (req, res) => {
+  req.body.userId = req.user.id;

```

Comment:
I don't see this as a good practice to modify request body, though. It's possible for other middleware to also have access to the `req` object, thereby having that information exposed to them.

Reply:
Thanks, @ezeilo-su , I will remove it from request body